src/lib.rs

```diff
@@ -192,21 +181,42 @@ impl Pixels {
         let mut encoder = self
             .device
             .create_command_encoder(&wgpu::CommandEncoderDescriptor { label: None });
-        for renderer in self.renderers.iter_mut() {
-            renderer.resize(&mut encoder, width, height);
-        }
-
-        self.queue.borrow_mut().submit(&[encoder.finish()]);
+        self.scaling_renderer
+            .resize(&mut self.device, &mut encoder, width, height);
+        self.queue.submit(&[encoder.finish()]);
     }
 
-    /// Draw this pixel buffer to the configured [`SurfaceTexture`].
-    ///
-    /// This executes all render passes in sequence. See [`RenderPass`].
+    /// Provides a [`wgpu::CommandEncoder`], a [`wgpu::TextureView`] from the swapchain which you can use to render to the screen, and the default [`ScalingRenderer`].
```

Comment:
Please split long lines so they do not cross 100 characters. I would also like to keep the initial doc line that describes succinctly what this method does.

Reply:
Sure.